packages/vue/src/components/molecules/SfSearchBar/SfSearchBar.vue

```diff
@@ -34,6 +34,10 @@ export default {
       type: [Number, String],
       default: null,
     },
+    iconSize: {
+      type: String,
+      default: "",
```

Comment:
default should follow default value in `sficon`

Reply:
there is ```default: ""``` in size in SfIcon :)